index.bs

```diff
@@ -1863,7 +1872,8 @@ BrowsingContextResult = (
   browsingContext.CreateResult /
   browsingContext.GetTreeResult /
   browsingContext.NavigateResult /
-  browsingContext.PrintResult
+  browsingContext.PrintResult /
```

Comment:
As a sidenote: don't we have to use `//` here as well?

Reply:
See https://github.com/w3c/webdriver-bidi/pull/512